cryptopunkscc/astrald
# AppHost client for android

Kotlin library module with apphost client for android.

* Network interface for apphost API
* TCP implementation of Network interface
* Basic binary serialization
* Json serialization
* Extension helpers

How to test manually:
1. Start astral node on localhost.
1. Change directory to `mobile/node/` and run `./gradlew client:test`. Test will check apphost methods by registering port on local node, connecting to it and sending short message.
